examples/summarize/src/summarize.py

```diff
@@ -0,0 +1,50 @@
+import os
+from airtop import AsyncAirtop, SessionConfigV1
+from dotenv import load_dotenv
+
+load_dotenv()
+
```

Comment:
This script does not run, it throws this error because it is missing the function definition

![image](https://github.com/user-attachments/assets/f5f37e1f-728e-402a-9ad9-db0419ccc401)

Add `async def run():` here and `asyncio.run(run())` at the end of the file or not sure if this is possibly to run without the await like the `examples/simple-interactions` on this project did without causing issues.

Reply:
I thought it was a problem with my setup
I copied the code you sent me, there existed these awaits
I saw the error, but I figured it was best to copy what you sent me
I changed the method from an Async to a Airtop
It's throwing an error I don't know how to debug

"Unable to create window"

I commited the code so you can see

Reply:
Your issue why the current code is that you are missing the https in the URL. Changing to `TARGET_URL = "https://www.nba.com"` will fix your code.

![image](https://github.com/user-attachments/assets/ffd2dc0b-0b8b-4a36-80f7-5a0f9b624903)